internal/rest/resources/resources.go

```diff
@@ -95,6 +95,15 @@ func ValidateEndpoints(extensionServers map[string]rest.Server, coreAddress stri
 			serverAddresses[server.Address.String()] = true
 		}
 
+		// Reserve the /core namespace for Microcluster
```

Comment:
I have the feeling we use the string `microcluster` in lowercase all across the repos now.